readme.md

```diff
@@ -5,18 +5,18 @@
 The Guardian's scala-based deployment system is designed to automate deploys by providing a web application that 
 performs and records deploys, as well as providing various integration points for automating deployment pipelines.
 
+## Documentation
+Riff-Raff's documentation is available in the application (under the Documentation menu) but can also be viewed under
+[riff-raff/public/docs](riff-raff/public/docs) in GitHub.
+
 ## Requirements
-Riff-Raff and Magenta have been built with the tools we use at the Guardian
```

Comment:
I think mentioning these as two distinct things here will be confusing for most people.